[3.2] docs(radiogroup): Revamped RadioGroup Overview (#825)
telerik/blazor-docs: components/radiogroup/appearance.md

> You can increase or decrease the size of the button by setting the `Size` parameter to a member of the `Telerik.Blazor.ThemeConstants.Button.Size` class:

Reviewer: ```suggestion
Change the size of the radio buttons by setting the `Size` parameter to a member of the `Telerik.Blazor.ThemeConstants.Button.Size` class:
```
